deploy.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+packages="domain auth backend"
+for package in $packages; do
+  cd $package
+  echo "😎 Deploying '$package'"
+  sls deploy
```

Comment:
Perhaps it would be better to use `npx sls deploy` here since some folks (myself included) like to install Serverless Framework locally to use different versions of the Framework in different projects. However there is a caveat: If you don't have the `serverless` NPM module installed when running `npx sls deploy`, NPM will prompt you to install a module named `sls`. So to be on the safe side, I'd suggest using `npx serverless deploy`. This should hopefully cover all bases.

Reply:
`npx serverless deploy` great suggestion!

I actually looked into this exact same thing (`sls` vs `serverless` package) a few weeks ago and I was questioning how `npx` resolves packages and binaries, only to find out that `sls` is an entirely different package and that `npx` will just use whatever bin is already in your system (and if there isn't one, just install the one with the exact name!). 😓